vlib/v/fmt/tests/struct_embed_input.vv

```diff
@@ -5,8 +5,8 @@ struct Foo {
 struct Test {}
 
 struct Bar {
+	Foo Test
```

Comment:
I think that a single `_keep.vv` file will be better than a pair of `_input.vv` and `_expected.vv` here.
`Foo Test` was very confusing for me for a minute, until I saw the _expected.vv file (which github decided to hide initially...)

Reply:
apparently a 82 byte text file is now too large for github 😕

Reply:
Will do it